config/path.php

```diff
@@ -0,0 +1,10 @@
+<?php
+return [
+	/*
+    |--------------------------------------------------------------------------
+    | Path 
+    |--------------------------------------------------------------------------
+    |
+    */
+    'path' => 'backend/images/',
```

Comment:
[rank C] You make me so confuse about this configuration (name & value). Why did you separate 2 files config with similar meaning. (path.php & upload.php)